<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValveDialog</class>
 <widget class="QDialog" name="ValveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ValveDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve16">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbValve13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V13</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
